templates/publications.rkt

```diff
@@ -35,3 +49,10 @@
     #:author (list ben-greenman zeina-migeed)
     #:href "https://popl18.sigplan.org/event/pepm-2018-on-the-cost-of-type-tag-soundness"
     #:venue pepm-2018))
+
+(define kas-scheme-2018
```

Comment:
these should go on the main page too (`index.rkt`)

Reply:
done.